woocommerce/woocommerce-gateway-stripe
# fix: payment gateways asset path

# Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/2046

The path in the payment gateways asset file is wrong, so it's not being included correctly.
Fixing the path, so that the dependencies are correctly queued, so that there's no error in the console:
![console error](https://user-images.githubusercontent.com/33387139/136426115-f25736df-2669-4203-821a-369673e36791.png)


# Testing instructions

- Ensure you have the `_wcstripe_feature_upe_settings` flag disabled
- Ensure you have the Stripe gateway enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- No more error in the console!
